_pages/manuals/rclone-config.md

```diff
@@ -25,9 +25,8 @@ Now walk through the questions. For SURFdrive you need the following information
 
 1. Type `n` for New remote
 2. For 'name' choose e.g. `surfdrive` (or SD)
-3. Choose option `41`for WebDAV&#x20;
-4. Fill in the URL. You can look this up via the web portal of SURFdrive. In the top right corner click your account name. Go to settings. In the left panel, click security. At the very bottom of the page there will be a section WebDAV passwords. There is also an URL, something like: [https://surfdrive.surf.nl/files/remote.php/nonshib-webdav](https://surfdrive.surf.nl/files/remote.php/nonshib-webdav)\
-   &#x20;Use this URL.
+3. Choose option `46`for WebDAV (this could differ depending on your Rclone version);
```

Comment:
If the number keeps changing, perhaps the wording of @RoelBrouwer makes more sense than putting the number in?

Reply:
changed